tracing/trace_keyspace_helper.hh

```diff
@@ -51,15 +51,14 @@ namespace tracing {
 
 class trace_keyspace_helper final : public i_tracing_backend_helper {
 public:
-    static const sstring KEYSPACE_NAME;
-    static const sstring SESSIONS;
-    static const sstring SESSIONS_TIME_IDX;
-    static const sstring EVENTS;
+    static constexpr std::string_view KEYSPACE_NAME = "system_traces";
```

Comment:
Why did you use in this patch constexpr instead of constinit as in the previous patch? I'm afraid I didn't understand the difference.

Reply:
In the previous patch, the data was defined out-of-line, so it was subject to link-order dependency.

In this patch, the data is defined inline, and so there is no link-order dependency, since the translation unit that uses the variable also sees its definition.